op-challenger/cmd/run_trace.go

```diff
@@ -38,3 +48,9 @@ var RunTraceCommand = &cli.Command{
 	Action:      cliapp.LifecycleCmd(RunTrace),
 	Flags:       runTraceFlags(),
 }
+
+var addMTCannonPrestate = &cli.StringFlag{
+	Name:    "add-mt-cannon-prestate",
```

Comment:
I don't think we need to make this MT specific, though it's not a bad starting point.  I'd like to be able to run the latest prestate from `develop` branch as well so ultimately I think we just have a CLI arg that provides additional cannon prestates to run. Just need to work out how to name them in metrics (possibly just use the prestate filename?)

Reply:
To be clear, I'm not against merging with this option as-is - just flagging that we'll almost certainly change it later to be more generic.  I'm not concerned with CLI stability for this tool though.